wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -267,9 +267,8 @@ static boolean isValidTrtLlmModelRepo(Path modelPath) throws IOException {
                     .forEach(
                             p -> {
                                 Path confFile = p.resolve("config.pbtxt");
-                                // Path tokenizer = p.resolve("1/tokenizer_config.json");
-                                // && Files.isRegularFile(tokenizer)
-                                if (Files.isRegularFile(confFile)) {
+                                Path tokenizer = p.resolve("tokenizer_config.json");
```

Comment:
Where do we put the tokenizer file? in 1 folder or the same location as config.pbtxt?

Reply:
same location as config.pbtxt